imports/plugins/included/inventory/server/hooks/hooks.js

```diff
@@ -112,9 +112,10 @@ function markInventoryShipped(doc) {
 function markInventorySold(doc) {
   const orderItems = doc.items;
   const cartItems = [];
+  // If a cartItemId exists it's a legacy order and we use that
   for (const orderItem of orderItems) {
     const cartItem = {
-      _id: orderItem.cartItemId,
+      _id: orderItem.cartItemId || orderItem._id,
```

Comment:
Is this change / behaviour documented elsewhere? Should be added to release docs?